app/src/main/kotlin/com/hippo/ehviewer/ui/reader/ReaderScreen.kt

```diff
@@ -54,45 +73,82 @@ fun ReaderScreen(info: BaseGalleryInfo, page: Int = -1) {
             pageLoader.stop()
         }
     }
+    val showSeekbar by Settings.showReaderSeekbar.collectAsState()
+    var controlledLazyList by remember { mutableStateOf(false) }
+    var currentPage by remember { mutableIntStateOf(-1) }
+    val lazyListState = rememberLazyListState()
+    val zoomableState = rememberZoomableState(zoomSpec = ZoomSpec(maxZoomFactor = 3f))
+    if (controlledLazyList) {
+        LaunchedEffect(Unit) {
+            snapshotFlow { currentPage }.drop(1).collect { index ->
+                lazyListState.animateScrollToItem(index)
+            }
+        }
```

Comment:
A click will set `controlledLazyList` to false immediately, which will cancel the sroll.

Reply:
Done